src/ui/connection_profiles.ui

```diff
@@ -2516,7 +2543,10 @@
                </size>
               </property>
               <property name="toolTip">
-               <string>Game description</string>
+               <string>Game description or your notes</string>
```

Comment:
will these be translatable/translated?

Reply:
Yes, ui texts are automatically included in translation process.
For example, this line on Crowdin in XX language is here:
https://crowdin.com/translate/mudlet/137/en-de#q=Game+description

Reply:
ok cool, I just wanted to double check at the front end of adding all the necessary labels rather than the back end.